src/components/inspectors/DateTimeExpression.vue

```diff
@@ -8,6 +8,7 @@
       class="p-0"
       :value="convertFromUTC(value)"
       @input="emitValue"
+      data-test="date-picker"
```

Comment:
This seemed to be the easiest solution to selecting this element in a test. Let me know if there's a better way.